types/react-fa/index.d.ts

```diff
@@ -37,15 +37,7 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-
-    // Does not work with TypeScript 2.3:
-    // Type '{ size: "lg"; children: Element[]; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<IconStackProps, ComponentState>> & Reado...'.
-    //   Type '{ size: "lg"; children: Element[]; }' is not assignable to type 'Readonly<IconStackProps>'.
-    //     Types of property 'children' are incompatible.
-    //       Type 'Element[]' is not assignable to type 'IconProps[]'.
-    //         Type 'Element' is not assignable to type 'IconProps'.
-    //           Property 'name' is missing in type 'Element'.
-    // children?: IconProps[];
+    children?: JSX.Element[];  // All JSX expression has type JSX.Element
```

Comment:
Correct. that would end up support both of these cases:

The normal use case
```ts
<IconStack size='lg'>
  <Icon name='circle' stack='2x' />
  <Icon name='flask' stack='1x' inverse />
</IconStack>
```

The not so normal, but technically supported use case
```ts
<IconStack size='lg'>
  <Icon name='flask' stack='1x' />
</IconStack>
```